bindinfo/session_handle_test.go

```diff
@@ -0,0 +1,540 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bindinfo_test
+
+import (
+	"context"
+	"strconv"
+	"testing"
+	"time"
+
+	"github.com/pingcap/parser/auth"
+	"github.com/pingcap/tidb/bindinfo"
+	"github.com/pingcap/tidb/errno"
+	"github.com/pingcap/tidb/metrics"
+	plannercore "github.com/pingcap/tidb/planner/core"
+	"github.com/pingcap/tidb/testkit"
+	"github.com/pingcap/tidb/util"
+	"github.com/pingcap/tidb/util/stmtsummary"
+	dto "github.com/prometheus/client_model/go"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGlobalAndSessionBindingBothExist(t *testing.T) {
```

Comment:
If these tests are run in serial, please name the file "xxx_serial_test.go"

Reply:
@w169q169 "these tests" mean all of the tests in this file, not only the commented one.

The rationale is that we'd prefer to run test in parallel for efficiency, and in order to help contributors be aware of whether a bunch of tests run in parallel or not, make serial test secondary and move then all in "xxx_serial_test.go"

Reply:
Okay, I got it. I make it wrong in the previous commit and I have fixed it.